scripts/validation-tests

```diff
@@ -67,11 +79,13 @@ run_tests()
         -e "AMAZON_SECRETKEY=${AMAZON_SECRETKEY}" \
         -e "AMAZON_ACCESSKEY=${AMAZON_ACCESSKEY}" \
         -v $(pwd):/source \
+        -v /opt/cattle/validation/:/target \
```

Comment:
Where does /opt/cattle/validation come from? Does it need to be present to bind mount it in?